pkg/yurthub/proxy/remote/loadbalancer.go

```diff
@@ -297,7 +297,7 @@ func (lb *loadBalancer) modifyResponse(resp *http.Response) error {
 				resp.Body = filterRc
 				if size > 0 {
 					resp.ContentLength = int64(size)
-					resp.Header.Set("Content-Length", fmt.Sprint(size))
+					resp.Header.Set(hubutil.HttpHeaderContentLength, fmt.Sprint(size))
```

Comment:
Use util.HttpHeaderContentLength uniformly, and put common variables under the util package to reduce the coupling of different projects